pystac/summaries.py

```diff
@@ -118,18 +125,7 @@ class Summarizer:
     summaryfields: Dict[str, SummaryStrategy]
 
     def __init__(self, fields: Optional[str] = None):
-        fieldspath = fields or FIELDS_JSON_URL
-        try:
-            jsonfields = _get_fields_json(fieldspath)
-        except Exception as e:
-            if fields is None:
```

Comment:
This special error was only for the case where pulling the default remote file failed.